src/server/views/transition-page.ejs

```diff
@@ -41,8 +41,23 @@
         </div>
     </div>
     <script>
+        (function (i, s, o, g, r, a, m) {
+            i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+                (i[r].q = i[r].q || []).push(arguments)
+            }, i[r].l = 1 * new Date(); a = s.createElement(o),
+                m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+        })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+        ga('create', '<%- gaTrackingId %>', 'auto');
+
+        // Logs transition-page loaded event to GA.
+        ga('send', 'event', '<%- gaEventType %>', '<%- gaOnLoad %>');
+
         function proceedToDestination() {
-            window.location = "<%- escapedLongUrl %>" // Use double-quotes to avoid templating issue as per RFC3986 (https://tools.ietf.org/html/rfc3986#appendix-C)
+            // Logs transition-page proceeded event to GA.
+            ga('send', 'event', '<%- gaEventType %>', '<%- gaOnProceed %>');
```

Comment:
I think we should be using the latest `gtag.js` instead - the code itself is [quite similar](https://developers.google.com/analytics/devguides/collection/gtagjs/migration).